packages/rulesets/src/spectral/test/path-for-tracked-resource-types.test.ts

```diff
@@ -0,0 +1,670 @@
+            responses: {
+              "200": {
+                schema: {
+                  $ref: "#/definitions/FooResource",
+                },
+              },
+            },
+          },
+        },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachine/{vmName}/providers/Microsoft.Billing/extensions/{extensionName}":
+        {
+          get: {
+            operationId: "Foo_Update",
+            description: "Test Description",
+            parameters: [
+              {
+                name: "foo_patch",
+                in: "body",
+                schema: {
+                  $ref: "#/definitions/FooRequestParams",
+                },
+              },
+            ],
+            responses: {
+              "200": {
+                schema: {
+                  $ref: "#/definitions/FooResourceUpdate",
+                },
+              },
+            },
+          },
+          put: {
+            operationId: "Foo_Update",
+            description: "Test Description",
+            parameters: [
+              {
+                name: "foo_patch",
+                in: "body",
+                schema: {
+                  $ref: "#/definitions/FooRequestParams",
+                },
+              },
+            ],
+            responses: {
+              "200": {
+                schema: {
+                  $ref: "#/definitions/FooResource",
+                },
+              },
+            },
+          },
+        },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/virtualMachine/{vmName}/providers/Microsoft.Billing/extensions/{extensionName}":
+        {
+          get: {
+            operationId: "Foo_Update",
+            description: "Test Description",
+            parameters: [
+              {
+                name: "foo_patch",
+                in: "body",
+                schema: {
+                  $ref: "#/definitions/FooRequestParams",
+                },
+              },
+            ],
+            responses: {
+              "200": {
+                schema: {
+                  $ref: "#/definitions/FooResourceUpdate",
+                },
+              },
+            },
+          },
+          put: {
+            operationId: "Foo_Update",
+            description: "Test Description",
+            parameters: [
+              {
+                name: "foo_patch",
+                in: "body",
+                schema: {
+                  $ref: "#/definitions/FooRequestParams",
+                },
+              },
+            ],
+            responses: {
+              "200": {
+                schema: {
+                  $ref: "#/definitions/FooResource",
+                },
+              },
+            },
+          },
+        },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/{vmName}/providers/Microsoft.Billing/extensions/{extensionName}":
```

Comment:
>Microsoft.Compute/{vmName} [](http://example.com/codeflow?start=84&length=26)

Does this mean, we can have resource staright away, without RT?

Reply:
same for this 
We are checking that scenario's in other rule in "verifyArmPath" function, this test is especially not to find errors if the resource is not tracked.